game/game.py

```diff
@@ -277,5 +276,10 @@ def is_game_over(self):
         return (False, GAME_OVER_NOT_MET)
 
     def get_current_player_name(self) -> str:
-        if self.remaining_moves > 0:
+        if self.remaining_moves > 0 and self.game_is_active and self.all_player_have_characters():
             return self.current_player.user_name
+        else:
+            return ''
+
+    def all_player_have_characters(self):
+        return self.player_1.characters and self.player_2.characters
```

Comment:
Can we have more than 2 players? If the answer is yes, then we'll need to loop through the players.

Reply:
No, we can't play with more than two players. Wumpus is only two players game